src/user-guide/task-implementation/task-run-modes.rst

```diff
@@ -0,0 +1,127 @@
+.. _task-run-modes.skip:
+
+Skip Mode
+=========
+
+.. versionadded:: 8.4.0
+
+   * Before 8.4.0 run modes could only be set for the workflow, not
+     for individual tasks.
+   * Skip mode.
+
+Skip mode is designed as an aid to workflow control:
+
+* It allows creation of :term:`dummy tasks<dummy task>` as part of workflow
+  design.
+* It allows skipping of tasks in a running workflow using
+  ``cylc broadcast -s 'run mode = skip'``.
+
+.. note::
+
+   Setting :cylc:conf:`[runtime][<namespace>]run mode` in your ``flow.cylc``
+   will lead to ``cylc validate`` returning a warning::
+
+      WARNING - The following tasks are set to run in skip mode:
+        * example_task
+
+   This is designed to prevent users running a task in skip mode by mistake.
+   If you are using skip mode deliberately then this can be
+   ignored.
+
+Task Outputs
+^^^^^^^^^^^^
+
+Skip mode allows the user to specify which task outputs
+will be emitted using :cylc:conf:`[runtime][<namespace>][skip]outputs`.
+
+By default:
+
+* All required outputs will be generated.
+* Succeeded will be generated if success is optional.
+* If outputs is specified and does not include either succeeded
+  or failed then succeeded will be generated.
+
+The outputs submitted and started are always generated and do not
+need to be defined in outputs.
+
+Task Event Handlers
+^^^^^^^^^^^^^^^^^^^
+
+By default task event handlers are disabled by skip mode, but they
+can be enabled using
+:cylc:conf:`[runtime][<namespace>][skip]disable task event handlers`.
+
+.. note::
+
+   Skip mode only works on live workflows. Workflows run in
+   :ref:`workflow-run-modes.simulation`
+   and :ref:`workflow-run-modes.dummy` ignore the
+   :cylc:conf:`[runtime][<namespace>]run mode` setting.
+
+
+Set task to succeeded
+^^^^^^^^^^^^^^^^^^^^^
+
+.. admonition:: Scenario
+
+   We don't need a task to run, but want to set it to succeeded.
+
+Broadcast :cylc:conf:`[runtime][<namespace>]run mode` setting the
+value to ``skip``.
```

Comment:
Or can use `cylc set --out skip`?

Reply:
This use case needs fleshing out, we're not actually interested in how to set a task to succeeded here.

The use case is along the lines of, I want to turn a task off in my workflow, or in a particular cycle of my workflow. So I don't want to set the output ahead of time (`cylc set --out=succeeded` or skip or whatever) because that would cause downstream triggering (although I could use the `--wait` option...), so I configure the task not to run instead.